optuna/visualization/_contour.py

```diff
@@ -55,17 +61,21 @@ def objective(trial):
 
     Args:
         study:
-            A :class:`~optuna.study.Study` object whose trials are plotted for their objective
-            values.
+            A :class:`~optuna.study.Study` object whose trials are plotted for their target values.
         params:
             Parameter list to visualize. The default is all parameters.
+        target:
+            A function to specify the value to display. If it is :obj:`None`, the objective values
+            are plotted.
+        target_name:
+            Target's name to display on the axis label and the legend.
```

Comment:
A nit, but `target_name` is only shown in the [color bar](https://plotly.com/python/contour-plots/#color-bar-title) title in case of the contour plots.

![image](https://user-images.githubusercontent.com/3255979/101453840-4d8c6180-3973-11eb-95bf-d69167879282.png)

```suggestion
            Target's name to display on the color bar.
```